### ISSUE TITLE
How to use pretrained backbone from torchvision? Loss became NaN..

### ISSUE THREAD

**Issue by jinxuan777** [facebookresearch/detectron2#183]
## ❓ Questions and Help
Help:
After rename weight, I use pretrained backbone from torchvision. But Loss became NaN.. Maybe something is wrong.

What I have done: 
1. Rename "bn[0-9]" to "conv[0-9].norm"
2. Rename "bn" to "norm"
3. Rename "conv1" to "stem.conv1"
4. Rename "layerN" to "res(N+1)" 
5. Added "^" to "backbone.bottom_up."

When load from pretrained backbone on training:
1. Some model parameters are not in the checkpoint:
backbone.bottom_up.res[2-5].0.shortcut.weight
backbone.bottom_up.res[2-5].0.shortcut.norm.{weight, bias}
2. The checkpoint contains parameters not used by the model:
  backbone.bottom_up.res2.0.downsample.1.{bias, running_var, running_mean, weight}
  backbone.bottom_up.res2.0.downsample.0.weight
  backbone.bottom_up.res3.0.downsample.1.{bias, running_var, running_mean, weight}
  backbone.bottom_up.res3.0.downsample.0.weight
  backbone.bottom_up.res4.0.downsample.1.{bias, running_var, running_mean, weight}
  backbone.bottom_up.res4.0.downsample.0.weight
  backbone.bottom_up.res5.0.downsample.1.{bias, running_var, running_mean, weight}
  backbone.bottom_up.res5.0.downsample.0.weight

How do I fix this issue?

**Comment by ppwwyyxx** [comment]
You're running training with your custom code rather than detectron2's models so we do not help on such questions. As said in the issue template:

> If you expect the model to work better, only in one of the two conditions we will help with it: (1) You're unable to reproduce the results documented in detectron2 model zoo. (2) It indicates a detectron2 bug.

The weights are definitely wrong from the logs you've seen.
